### DISCUSSION TITLE
Unused zombies

### DISCUSSION THREAD

**Discussion by simPod** [composer-unused/composer-unused#188] [Ideas]
I have discovered few packages listed in my `unused` which are no longer required.

Would it make sense to also report these as invalid config?

Example: Consider a package `v/lib` in `require` and also in `unused` sections. After some time it gets removed from `require` section but not from `unused`.

**Comment by icanhazstring** [comment]
Hi, that is actually a good idea. Giving a warning that there are packages declared ignored for unused when they are no longer required.

I'll take it as a new feature 👍
